startup/gui/menus.py

```diff
@@ -543,6 +543,7 @@ def __usdLightCreator( lightType ) :
 
 nodeMenu.append( "/USD/Attributes", GafferUSD.USDAttributes, searchText = "USDAttributes" )
 nodeMenu.append( "/USD/Layer Writer", GafferUSD.USDLayerWriter, searchText = "USDLayerWriter" )
+nodeMenu.append( "/USD/Promote Instances", GafferUSD.PromoteInstances, searchText = "PromoteInstances" )
```

Comment:
Following on from my earlier comment, should the node be called PromotePointInstances, or is that too verbose?